lib/plurimath/math/function/g.rb

```diff
@@ -10,6 +10,10 @@ def initialize(value)
           @value = value
         end
 
+        def to_asciimath
+          "g#{value&.to_asciimath}"
```

Comment:
Seems that there are many instances of `class Function` subclasses that has this method. Can we use a method in Function like this:
```ruby
def to_asciimath
  "#{function_name_in_asciimath}#{value.to_asciimath}"
end
```